NativeAuthSampleApp/ProtectedAPIViewController.swift

```diff
@@ -87,23 +92,45 @@ class ProtectedAPIViewController: UIViewController {
         accountResult?.signOut()
 
         accountResult = nil
-        
-        accessToken = nil
+        accessTokenAPI1 = nil
+        accessTokenAPI2 = nil
 
         showResultText("Signed out")
-
         updateUI()
     }
     
-    @IBAction func protectedAPIPressed(_: Any) {
-        guard let accessToken = self.accessToken else {
-            let errorMessage = "No access token found"
-            print(errorMessage)
-            showResultText(errorMessage)
+    @IBAction func protectedApi1Pressed(_: Any) {
+        guard   let url = protectedAPIUrl1,
+                !protectedAPIScopes1.isEmpty else {
+            showResultText("API 1 not configured.")
+            return
+        }
+        
+        if let accessToken = accessTokenAPI1 {
+            accessProtectedAPI(apiUrl: url, accessToken: accessToken)
+        } else {
+            accountResult?.getAccessToken(scopes: scopesApi1, delegate: self)
```

Comment:
this should be "protectedAPIScopes1"
```suggestion
            accountResult?.getAccessToken(scopes: protectedAPIScopes1, delegate: self)
```

Reply:
Fixed.